### DISCUSSION TITLE
Add "not exists" to the "withAggregate" method

### DISCUSSION THREAD

**Discussion by MadZee-M** [laravel/framework#48879] [Ideas]
Good day! I would like to suggest adding "not exists" to the "withAggregate" method.
I looked at the method. You can achieve this by adding an exception for “not exists” as well (currently “exists” already has an exception).
Thank you!

_There are three lines highlighted (635, 669 and 671) in which I made changes to achieve success._
![Screenshot_1](https://github.com/laravel/framework/assets/32099597/0237cdad-cb36-4830-8668-f1c27f024e39)

**Comment by macropay-solutions** [comment]
Nice catch. Can you give a scenario in which this would be used?
